compose-lodestar.yml

```diff
@@ -0,0 +1,87 @@
+# Docker compose file to aid developers to test lodestar VCs in a distributed validator cluster.
+version: "3.8"
+
+services:
+  #  _           _           _             
+  # | | ___   __| | ___  ___| |_ __ _ _ __ 
+  # | |/ _ \ / _` |/ _ \/ __| __/ _` | '__|
+  # | | (_) | (_| |  __/\__ \ || (_| | |   
+  # |_|\___/ \__,_|\___||___/\__\__,_|_|   
+  vc0-lodestar:
+    image: nflaig/lodestar:dvt-compatible
+    entrypoint: /opt/lodestar/run.sh
+    networks: [ cluster ]
+    depends_on: [ node0 ]
+    restart: unless-stopped
+    environment:
+      BEACON_NODE_ADDRESS: http://node0:3600
+      ETH2_NETWORK: ${ETH2_NETWORK:-goerli}
+    volumes:
```

Comment:
I would recommend to add a persistent volume for `dataDir`, this can be specified by setting `--dataDir` lodestar cli option, see [Lodestar CLI docs](https://chainsafe.github.io/lodestar/reference/cli/#global-options) for further reference